src/profile-logic/marker-schema.js

```diff
@@ -678,6 +678,18 @@ export function markerPayloadMatchesSearch(
         payloadField.format === 'flow-id' ||
         payloadField.format === 'terminating-flow-id'
       ) {
+        if (typeof value !== 'number') {
+          console.error(
+            `In marker ${marker.name}, the key ${payloadField.key} has an invalid value ${value} as a unique-string.`
+          );
```

Comment:
I think we need to distinguish about these 2 cases:
* the value is missing (value === undefined) => this can happen and is not an error, therefore there shouldn't be a console.error output
* the value is present but is not a number => this is an error and outputting an error to the console sounds right.